tests/framework/installer/ceph_manifests_v1.0.go

```diff
@@ -1303,13 +1303,14 @@ kind: StorageClass
 metadata:
    name: ` + storageClassName + `
 provisioner: ceph.rook.io/block
+allowVolumeExpansion: true
```

Comment:
ceph_manifests_v1.0.go should have only changes to the test interface, but not make changes to the manifests. Let's revert this storage class change. Only the upgrade integration test hits this code path anyway, so this won't be needed.